python3-job-type/job-template.Dockerfile

```
@@ -20,6 +22,22 @@ COPY . /src/job/
 RUN chmod -R a+rw /src/job/
 
 CMD python -u -m job_wrapper run "{{ manifest.python.entrypoint_path }}" "{{ manifest.python.entrypoint_class }}" < /dev/null
+{% endif %}
+
+{% if manifest.jobtype_extra %}
+{% if manifest.jobtype_extra.requirements_path %}
```

I think you can use your backwards compatible `get_jobtype_extra` and the above conditions won't be needed then
```suggestion
{% if manifest.get_jobtype_extra().requirements_path %}
```